### PR TITLE
Hotfix/update sheet and registry db with genebuild status

### PR DESCRIPTION
**Pull request by ndliberial**
# Requirements
When creating your Pull request, please fill out the template below:

# PR details
_Is this a fix/ update/ new feature?_
Update
_Include a short description_
This update allows the status, start date and genebuilder info to be registered in the registry database and the sheets once a Genebuild kictstarts
_Include links to JIRA tickets_
https://www.ebi.ac.uk/panda/jira/browse/ENSGENEBUI-502
https://www.ebi.ac.uk/panda/jira/projects/ENSGENEBUI/issues/ENSGENEBUI-508?filter=allopenissues&orderby=assignee+ASC%2C+priority+DESC%2C+updated+DESC
# Testing
_Have you tested it?_
Yes on creating a new Genebuild and updating the sheets
# Assign to the weekly GitHub reviewer
_If you are a member of Ensembl, please check the Genebuild weekly Rotas and assign this week's GitHub reviewer to the PR_
Kostas

### CODE REVIEW

**Comment by thibauthourlier** [comment]
@ndliberial how different is this PR from PR220?

**Comment by ndliberial** [comment]
Hi Thibaut, 
This PR is an update from PR 220. It has added function to retrieve clade via taxon_id during custom/early load of assembly. It also has some code refactoring implemented for comparing values between the sheets and the db.

**Comment by ndliberial** [comment]
@thibauthourlier 
This PR is an update from PR 220. It has added function to retrieve clade via taxon_id during custom/early load of assembly. It also has some code refactoring implemented for comparing values between the sheets and the db.

**Review by kbillis — CHANGES_REQUESTED** [review]
I think there are few places in python code that are not added during this commit comments but can be improved by aligning "#" lines to correct lines and other lines that could be removed, for example at the end of update_assembly_registry_sheet.py . Could you look at it?

File: scripts/genebuild/update_assembly_registry_sheet.py [hunk 1]
```diff
@@ -211,12 +211,13 @@ def update_assembly_sheet(assembly_db_data,meta_db_data,existing_sheet_records,a
...
       # Check if transcriptomic data status from db is null. If yes, check if assembly has been handed over or if assembly meets candidate assembly criteria
-      if rnaseq_data is None and sheet_rnaseq_data_val == 'Non candidate assembly' or sheet_rnaseq_data_val == 'No RNAseq data':
+      if ((rnaseq_data is None) and (sheet_rnaseq_data_val == 'Non candidate assembly' or sheet_rnaseq_data_val == 'No RNAseq data')):
         # Nothing to update
         print("No update on rnaseq data status for: " + gca)
-      elif rnaseq_data is None and annotation_status == 'Handed over': #It is possible to annotate a species and handover without RNASeq data
+      elif ((rnaseq_data is None) and (annotation_status == 'Handed over')): #It is possible to annotate a species and handover without RNASeq data
+
         # update the RNASeq data status
         rnaseq_data = 'Done'
```
Comment (kbillis) [inline scripts/genebuild/update_assembly_registry_sheet.py:222]: why rnaseq data done, when there is no rnaseq data?

Comment (ndliberial) [reply scripts/genebuild/update_assembly_registry_sheet.py]: Value changed to N/A as in this case we don't need to test for RNASeq data anymore

**Review by ndliberial — COMMENTED** [review]
All unwanted lines have been removed and comment blocks added in the right places in the updated commit

**Review by kbillis — APPROVED** [review]
Thanks for the changes.

**Pull request merged**